src/Sarif/Core/SarifRewritingVisitor.cs

```diff
@@ -1,85 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Linq;
-
-namespace Microsoft.CodeAnalysis.Sarif
-{
-    /// <summary>
-    /// Rewriting visitor for the Sarif object model.
-    /// </summary>
-    public abstract partial class SarifRewritingVisitor
```

Comment:
>SarifRewritingVisitor [](start = 34, length = 21)

yay! all dictionaries converted to arrays, no need for this key visiting thing. _unless_ we decide it's required in future for things like graphs dictionary